dbt/include/snowflake/macros/materializations/dynamic_table.sql

```diff
@@ -1,68 +1,38 @@
 {% materialization dynamic_table, adapter='snowflake' %}
 
-    {% set original_query_tag = set_query_tag() %}
+    {% set query_tag = set_query_tag() %}
 
     {% set existing_relation = load_cached_relation(this) %}
     {% set target_relation = this.incorporate(type=this.DynamicTable) %}
-    {% set intermediate_relation = make_intermediate_relation(target_relation) %}
-    {% set backup_relation_type = target_relation.DynamicTable if existing_relation is none else existing_relation.type %}
-    {% set backup_relation = make_backup_relation(target_relation, backup_relation_type) %}
 
-    {{ dynamic_table_setup(backup_relation, intermediate_relation, pre_hooks) }}
```

Comment:
Backup and intermediate relations are handled in `replace`, `create`, etc. so setup boils down to just running the hooks. The samae is true below for teardown.